### PR TITLE
Review sc - section 1.1.

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

File: 01-introduction.Rmd [hunk 1]
```diff
@@ -4,11 +4,21 @@
...
+
+```{block, json, type='rmdtip'}
+There are many places to find generic and domain specific R teaching materials. The first place to look is inside R itself. Entering `help.start()` into the R console will reveal some useful resources, not least the classic official guide *An Introduction to R*. Entering `vignette()` will display a list of guides packaged *within your R installation* (and hence free from the need of an internet connection). To see the vignette for a specific topic, just enter the vignette's name into the same command, e.g. `vignette(package = "dplyr", "introduction")` to see the introductory vignette for the **dplyr** package.
```
Comment (csgillespie) [inline 01-introduction.Rmd]: I like the idea of this note, but the official guide is tricky. When you teach, do you really recommend help.start() as the first port of call?

File: 07-performance.Rmd [hunk 2]
```diff
@@ -80,7 +80,7 @@ An additional quirk of `ifelse()` is that although it is more *programmer effici
 ```{r}
 marks = runif(n = 10e6, min = 30, max = 99)
```
Comment (csgillespie) [inline 07-performance.Rmd]: This is probably too picky, but should marks be rounded or generated via sample?

**Comment by csgillespie** [comment]
Duplicate label `json` makes travis fail

**Comment by Robinlovelace** [comment]
Not sure why it was failing after the json tag was fixed - any ideas? Completed the review up to C6.

**Pull request merged**